src/Entity/Consumer.php

```diff
@@ -93,35 +98,111 @@ public function getConsumerEmail(): ?string
     public function setConsumerEmail(string $consumer_email): self
     {
         $this->consumer_email = $consumer_email;
+        return $this;
+    }
+
+    /**
+     * @return Collection
+     */
+    public function getReviews(): Collection
+    {
+        return $this->reviews;
+    }
+
+    /**
+     * @param ArrayCollection $reviews
+     */
+    public function setReviews(ArrayCollection $reviews): void
+    {
+        $this->reviews = $reviews;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getGoogleId()
+    {
+        return $this->googleId;
+    }
+
+    /**
+     * @param mixed $googleId
+     */
+    public function setGoogleId($googleId): void
+    {
+        $this->googleId = $googleId;
+    }
+
+    public function getYes(): ?string
+    {
+        return $this->yes;
+    }
+
+    public function setYes(string $yes): self
+    {
+        $this->yes = $yes;
 
         return $this;
     }
 
-    public function getConsumerPassword(): ?string
+    /**
+     * A visual identifier that represents this user.
+     *
+     * @see UserInterface
+     */
+    public function getUsername(): string
     {
-        return $this->consumer_password;
+        return (string)$this->yes;
     }
 
-    public function setConsumerPassword(string $consumer_password): self
+    /**
+     * @see UserInterface
+     */
+    public function getRoles(): array
     {
-        $this->consumer_password = $consumer_password;
+        $roles = $this->roles;
+        // guarantee every user at least has ROLE_USER
+        $roles[] = 'ROLE_USER';
+
+        return array_unique($roles);
+    }
+
+    public function setRoles(array $roles): self
+    {
+        $this->roles = $roles;
 
         return $this;
     }
 
     /**
-     * @return Collection
+     * @see UserInterface
      */
-    public function getReviews(): Collection
+    public function getPassword(): string
```

Comment:
We still have `getConsumerPassword` method and fixtures are also kind a failing, so I guess we should choose this method? Once again please check admin panel since it is dependant from methods naming.